cleverhans/attacks_tf.py

```diff
@@ -29,12 +29,18 @@ def fgsm(x, predictions, eps, clip_min=None, clip_max=None):
                     value for components of the example returned
     :param clip_max: optional parameter that can be used to set a maximum
                     value for components of the example returned
+    :param y: the output placeholder. If doing adversarial training, this
+            should be none (the default).
```

Comment:
I'm confused by this comment---do you mean it should be None to avoid the label leaking effect? The reason I find the comment confusing is that we should use a non-leaking method at evaluation time as well as at training time